bundles/org.openhab.binding.modbus.kermi/README.md

```diff
@@ -0,0 +1,193 @@
+
+## Discovery
+
+There's no discovery.
+Modbus registers are available for all devices.
+
+## Device IDs
+
+| Device              | device ID | Comment                                          |
+|---------------------|-----------|--------------------------------------------------|
+| X-Center            | 40        | on cascade-circuits: slave1: 41, slave2: 42, ... |
+| StorageSystemModule | 50 or 51  |                                                  |
+
+## Thing Configuration
+
+The needed Things can be found in the **Modbus Binding** and have to be added manually without Discovery
+
+1. Create _Modbus TCP Bridge_ with matching Settings of your Kermi Device
+
+- IP Address
+- Device ID
+- Port ID
+
+1. Create _Kermi Heat Pump X-Center_ and attach it to the previous installed _Modbus TCP Bridge_. Configuration requires an approriate Data Refresh Interval with more than 2000 Milliseconds, default is 5000. If it's too fast, you may experience errors in openhab or your x-center ! (Reboot if x-center stops responding on network access). You can enable "PV Modulation" if you want to read the values (default: disabled)
+
+2. If you have a StorageSystemModule add _StorageSystemModule_ Thing, you have to create a separate _Modbus TCP Bridge_ because of the other device-id. (todo)
+
+
+### Modbus TCP Slave
+
+| Parameter | Type    | Description                                                   |
+|-----------|---------|---------------------------------------------------------------|
+| host      | text    | IP Address or reachable hostname of your device               |
+| port      | integer | TCP Port of your Kermi device Modbus Settings. Default is 502 |
+| deviceid  | integer | Modbus ID of your Kermi device Modbus Settings. Default is 40 |
+
+### Kermi Heat Pump X-Center
+
+Select as Bridge your previously created Modbus TCP Slave.
+
+| Parameter       | Type    | Description                                                             |
+|-----------------|---------|-------------------------------------------------------------------------|
+| refresh         | integer | Refresh Rate of x-center values in Milliseconds                         |
+
+### Kermi StorageSystemModule (todo)
+
+Select as Bridge a seperate (second) Modbus TCP Slave.
+
+| Parameter | Type    | Description                                                        |
+|-----------|---------|--------------------------------------------------------------------|
+| host      | text    | IP Address or reachable hostname of your device (same as x-center) |
+| port      | integer | TCP Port of your Kermi device Modbus Settings. Default is 502      |
+| deviceid  | integer | Modbus ID of your Kermi device Modbus Settings. Default is 50      |
+
+
+### Things
+
+```java
+Bridge modbus:tcp:device "Kermi X-Center Modbus TCP" [ host="xcenter", port=502, id=40 ] {
+ Bridge kermi-xcenter "Kermi X-Center Heat Pump" [ refresh=5000 ]
```

Comment:
The identifier of the Bridge is missing, for example "heatpump".
It should be:
Bridge kermi-xcenter **heatpump** "Kermi X-Center Heat Pump" [ refresh=5000 ]